src/pages/validator/amplifier/gmp.mdx

```diff
@@ -0,0 +1,95 @@
+# General Message Passing with Amplifier
+
+In this tutorial, you will learn how to execute a GMP call from the Ethereum Sepolia to the Avalanche Fuji testnets.
```

Comment:
I'd love some additional description about why this is so different than normal GMP, or why we are doing all of this manual work. Ideally we'd justify the additional complexity and talk about the benefits.

Reply:
it's not really different than normal GMP. These steps ultimately will be performed by the relayer, not the user. This tutorial is mainly for people building relayers.